router/xgress_edge_tunnel/fabric.go

```diff
@@ -517,12 +522,31 @@ type tunnelTerminator struct {
 	created       atomic.Bool
 	closeCallback concurrenz.AtomicValue[func()]
 	closed        atomic.Bool
+	notifyCreated chan struct{}
 }
 
 func (self *tunnelTerminator) SendHealthEvent(pass bool) error {
 	return self.provider.sendHealthEvent(self.id, pass)
 }
 
+func (self *tunnelTerminator) NotifyCreated() {
+	select {
+	case self.notifyCreated <- struct{}{}:
+	default:
+	}
+}
+
+func (self *tunnelTerminator) WaitForCreated(timeout time.Duration) bool {
+	if self.created.Load() {
+		return true
+	}
+	select {
+	case <-self.notifyCreated:
+	case <-time.After(timeout):
```

Comment:
Probably doesn't need a log because it's just the timeout until this attempt fails. The attempt will be re-retried. So as a 'temporary' failure, I don't think it needs a log.

Reply:
Fixed